os/se/ameba/Make.defs

```diff
@@ -41,3 +44,17 @@ CFLAGS += -I$(TOPDIR)/board/rtl8721csm/src/component/soc/realtek/amebad/misc
 CFLAGS += -I$(TOPDIR)/board/rtl8721csm/src/component/os/os_dep/include
 CFLAGS += -I$(TOPDIR)/board/rtl8721csm/src/component/common/network/ssl/ssl_ram_map/rom
 endif
+
+ifeq ($(CONFIG_AMEBALITE_TRUSTZONE),y)
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/mbed/hal_ext/
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/soc/amebalite/cmsis
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/soc/amebalite/swlib
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/soc/amebalite/app/monitor/include
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/soc/amebalite/misc
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/project/realtek_amebaLite_va0_example/inc/inc_km4
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/soc/amebalite/fwlib/include
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/ssl/mbedtls_ram_map/rom
+CFLAGS += -I$(TOPDIR)/board/rtl8720e/src/component/os/os_dep/include
```

Comment:
Could you confirm they all are necessary indeed?

Reply:
Yes, these are necessary